src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/SQLFunctions.java

```diff
@@ -620,6 +665,69 @@ private String upper(String property, String culture) {
         return property + ".toUpperCase(Locale.forLanguageTag(\"" + culture + "\"))";
     }
 
+    private Tuple<String, String> length(SQLExpr field, String valueName) {
```

Comment:
Do you happen to know what's this `valueName` and why other functions like `locate`, `l/rtrim` don't have it?

Reply:
Actually the `nameName` is passed as null. I've deleted them.